### ISSUE TITLE
How do you hide files or folders?

### ISSUE THREAD

**Issue by whitevo** [Microsoft/vscode#39506]
- VSCode Version: Code 1.18.1 (929bacba01ef658b873545e26034d1a8067445e9, 2017-11-16T18:32:36.023Z)
- OS Version: Windows_NT x64 10.0.16299
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
Lua|key|0.0.9
theme-monokai-pro-vscode|mon|1.1.2

---

Steps to Reproduce:

1. right click on file or folder.
2. there is no option to hide file.

Reproduces without extensions: Yes

Is it possible?
would be nice If there is options to hide files or folders.
even better if I could save templates of hidden files and folders.
When file is hidden, it will go into folder like state "hidden" under where are all the folders with files which have been hidden. right click on them will have option to unhide

**Comment by sergeos** [comment]
You can define it in user preferences. Maybe this video wil help you:
https://www.youtube.com/watch?v=Q2kspPimO4A

**Comment by isidorn** [comment]
@sergeos thanks for answering
@whitevo this could also be made as an extension to add the context menu to help users hiding their folders

**Comment by whitevo** [comment]
I found an extension which supposes to do that, but it doesn't work at all:
https://marketplace.visualstudio.com/items?itemName=devine-davies.make-hidden

Then tried the Peep thing extension, but don't see the point in it, if it even works then all it does is disable change the true or false value right?

@isidorn 
took me 4 hours to figure out how to change space amount when pressing tab. I don't even want to think about making any more complicated changes here.

@sergeos 
Thank you, that helped, with this video I understood how to hide specific file names.
The if something is something, I did not understand and does not work for me.
